### ISSUE TITLE
Implement image viewer widget for waveform PVs 

### ISSUE THREAD

**Issue by GDYendell** [epics-containers/pvi#74]
And/or V4 NTNDArray PVs?

**Comment by jsouter** [comment]
I think this is ready to close

**Comment by jsouter** [comment]
Is this ready to close or is there more specific work that needs to be done for NTNDArrays?

**Comment by GDYendell** [comment]
Yeah it might work now, but I don't know if we have ever generated a screen and tested it against a live PV. Does PandA have a waveform image that uses this?
